tools/sgcollect_info

```diff
@@ -15,6 +15,8 @@ import uuid
 
 from sys import platform as _platform
 
+from botocore.vendored import requests
```

Comment:
Update to remove new import based on https://github.com/couchbase/sync_gateway/pull/4178#discussion_r298133190